r/R/filesystem.R

```diff
@@ -554,7 +554,8 @@ GcsFileSystem <- R6Class("GcsFileSystem",
       # Convert from nanoseconds to POSIXct w/ UTC tz
       if ("expiration" %in% names(out)) {
         out$expiration <- as.POSIXct(
-          out$expiration / 1000000000, origin = "1970-01-01", tz = "UTC"
+          out$expiration / 1000000000,
+          origin = "1970-01-01", tz = "UTC"
```

Comment:
I'm surprised that this wasn't produced: This is what it should be given linting definitions

```suggestion
          out$expiration / 1000000000,
          origin = "1970-01-01", 
          tz = "UTC"
```

Reply:
Thanks! I've applied.